src/Bootstrap/src/Autoconfig/Steeltoe.Bootstrap.Autoconfig.csproj

```
@@ -0,0 +1,60 @@
+    <ProjectReference Include="..\..\..\Configuration\src\ConfigServerCore\Steeltoe.Extensions.Configuration.ConfigServerCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\KubernetesCore\Steeltoe.Extensions.Configuration.KubernetesCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\PlaceholderCore\Steeltoe.Extensions.Configuration.PlaceholderCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Connectors\src\Connector.EF6Core\Steeltoe.Connector.EF6Core.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Connectors\src\ConnectorCore\Steeltoe.Connector.ConnectorCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Discovery\src\ClientCore\Steeltoe.Discovery.ClientCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Logging\src\DynamicSerilogCore\Steeltoe.Extensions.Logging.DynamicSerilogCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Management\src\CloudFoundryCore\Steeltoe.Management.CloudFoundryCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Management\src\EndpointCore\Steeltoe.Management.EndpointCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Management\src\KubernetesCore\Steeltoe.Management.KubernetesCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Management\src\TaskCore\Steeltoe.Management.TaskCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Management\src\TracingCore\Steeltoe.Management.TracingCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Security\src\Authentication.CloudFoundryCore\Steeltoe.Security.Authentication.CloudFoundryCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Security\src\Authentication.MtlsCore\Steeltoe.Security.Authentication.MtlsCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Security\src\DataProtection.CredHubCore\Steeltoe.Security.DataProtection.CredHubCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Security\src\DataProtection.RedisCore\Steeltoe.Security.DataProtection.RedisCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Connectors\src\CloudFoundry\Steeltoe.Connector.CloudFoundry.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Connectors\src\Connector.EFCore\Steeltoe.Connector.EFCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Logging\src\DynamicLogger\Steeltoe.Extensions.Logging.DynamicLogger.csproj" PrivateAssets="All" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\..\Connectors\src\ConnectorBase\**\*TypeLocator.cs" LinkBase=".\TypeLocators" />
```

```suggestion
    <Compile Include="..\..\..\Connectors\src\ConnectorBase\**\*TypeLocator.cs" LinkBase="TypeLocators" />
```

Not sure if this matters for non-Windows+VS scenarios, but the `.\` adds an extra folder named `.` for me which adds an unnecessary layer